src/humanDecision.ttl

```diff
@@ -18,7 +18,7 @@
                                                       <http://sweetontology.net/relaHuman> ,
                                                       <http://sweetontology.net/reprMathGraph> ,
                                                       <http://sweetontology.net/reprMathOperation> ;
-                                          rdfs:label "SWEET Ontology" ;
+                                          rdfs:label "Human decision process SWEET Ontology" ;
```

Comment:
Hi @smrgeoinfo thank you for the PR, can we make the Label capitalization consistent? e.g.

```rdfs:label "Human Decision Process SWEET Ontology" ;```

Note the capitalization on **D**ecision and **P**rocess

Reply:
I'll go with that-- all words in caps, and make changes

Reply:
new commit https://github.com/ESIPFed/sweet/pull/88/commits/06c1ee77e82c58c03081bf0bc50f290e41a5a148 added to pull request